raspi-config

```diff
@@ -1997,6 +1997,64 @@ do_network_install_ui() {
   fi
  }
 
+do_libliftoff() {
+  if [ "$INTERACTIVE" = True ]; then
+    RET=$(whiptail --title "Raspberry Pi Software Configuration Tool (raspi-config)" --menu "libliftoff Options" $WT_HEIGHT $WT_WIDTH $WT_MENU_HEIGHT \
+        "L1 Disable" "Disable libliftoff hardware overlays" \
+        "L2 Enable" "Enable libliftoff hardware overlays (experimental)" \
+        3>&1 1>&2 2>&3)
+  else
+    RET=$1
+    true
+  fi
+  if [ $? -eq 0 ]; then
+    case "$RET" in
+      L1*)
+        LIBLIFTOFF_ENABLED=0
+        STATUS="disabled"
+        ;;
+      L2*)
+        LIBLIFTOFF_ENABLED=1
+        STATUS="enabled"
+        ;;
+      *)
+        whiptail --msgbox "Programmer error, unrecognised libliftoff option" 20 60 2
+        return 1
+        ;;
+    esac
+
+    # Create the config dir and/or environment file, if needed
+    if [ -z $HOMEDIR ]; then
+      whiptail --msgbox "Unknown user home directory" 20 60 2
+      return 1
+    fi
+    LABWC_DIR="$HOMEDIR/.config/labwc"
```

Comment:
If the `.config` directory doesn't exist, then only the labwc directory would have the right permissions, no?

Reply:
My `chown` on line 2042 separately includes the .config and labwc directories, I'm pretty sure this works.

Reply:
Ah, you're absolutely right. Missed that you were passing multiple paths.